lapis-docs/src/config.ts

```diff
@@ -1,5 +1,7 @@
 import fs from 'fs';
 import { parse } from 'yaml';
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
Why do we need this? Actually vite should already make sure that .env files are loaded.

Reply:
this didnt work for me (I got variable undefined warnings)and @JonasKellerer had a comment in the code that there was an issue with this - adding dotenv fixed the issue for me locally- however I sadly think this is causing the e2e test failures